src/styles/adaptive.scss

```diff
@@ -0,0 +1,36 @@
+@media (max-width: 1023px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
```

Comment:
all styles for catalog should be in `catalog.scss`
don't create separate file for adaptive styles
all styles for all screens for one block should be in one file

Reply:
Sorry! I had some problems with 'deploy', because of it you coudn't see changes in branch 'develop'